### ISSUE TITLE
Screen Reader Enhancements

### ISSUE THREAD

**Issue by slannon97** [jitsi/jitsi-meet#10746]
Jitsi has a few issues when used with a screen reader. I'm totally blind and use the NVDA screen reader from www.nvaccess.org. It's about 90% usable which is fantastic, and many of these enhancements should be extremely easy and quick to implement. I've also linked to a zip file containing an mp3 recording where I demo Jitsi using NVDA on two computers. I used the public Jitsi server at meet.jit.si. This may help in terms of understanding what I'm talking about.

In order to reproduce these issues, download NVDA, run it, and use it to navigate a meeting with the arrow keys, tab or shift tab, and b and shift b to move forward and backward by buttons. Additionally, you may want to use h and shift h or s and shift s to navigate by headings and separators.

The buttons for muting/unmuting microphone, starting/stopping the camera, screen sharing, showing or hiding participants, etc should change their labels to reflect what's going to happen when activating that button. For example, mute button should be read when the microphone is unmuted and the resulting action will mute it, or unmute in the opposite case.

Anything that reports as clickable and can only be activated with the mouse should become standard button controls navigable with standard keyboard and screen reader commands such as space bar or enter. This means routing the mouse and left clicking is no longer necessary. Examples of this include the local and remote user control buttons, selecting a breakout room, etc.

The more menu items should be converted to buttons to make them easier to use.

Headings should be used to clearly divide the screen into sections to make it quick and easy to navigate.

ARIA or some other method should be used to automatically allow the screen reader to announce important events, such as when people join, leave, enter the lobby, or send messages to either the group chat or the individual. Currently, you hear a sound, but you must manually navigate to find out what happened.

When displaying the list of participants, there should be information accessible to the screen reader that indicates whether the user's microphone and camera are on or off. Please let me know 

if you need additional information. I'm not a programmer, but I consider myself a relatively advanced screen reader user, so please let me know if I can provide further assistance or information.[Download Jitsi Audio Demo](https://www.christopherw.me/files/Jitsi_Screen_Reader_Demo.zip)
